cmd/completion.go

```diff
@@ -40,19 +40,12 @@ fish:
 
   # To load completions for each session, execute once:
   $ containerlab completion fish > ~/.config/fish/completions/containerlab.fish
-
-PowerShell:
-
-  PS> containerlab completion powershell | Out-String | Invoke-Expression
-
-  # To load completions for every new session, run:
-  PS> containerlab completion powershell > containerlab.ps1
-  # and source this file from your PowerShell profile.
 `,
 	DisableFlagsInUseLine: true,
 	ValidArgs:             []string{"bash", "zsh", "fish"},
 	Args:                  cobra.ExactValidArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Root().MarkFlagFilename("topo", "*.yaml", "*.yml")
```

Comment:
I also think that it is best to add this line not in the completions file, but in the file where `topo` flag is defined

Reply:
ok, moved back to root.go